vite.config.ts

```diff
@@ -1,8 +1,9 @@
 import { sveltekit } from "@sveltejs/kit/vite";
 import { defineConfig } from "vitest/config";
+import { compileProtobuffers } from "./compileProtobuffers";
 
 export default defineConfig({
-	plugins: [sveltekit()],
+	plugins: [sveltekit(), compileProtobuffers],
```

Comment:
```suggestion
	plugins: [compileProtobuffers, sveltekit()],
```
You should compile protobuffers before activating sveltekit.

You can just commit this suggestion, you don't have to write it yourself.